test_programs/plonky2_verify_success/non_first_public_input/VerifierTest.toml

```diff
@@ -1,5 +1,5 @@
 # This file is separate from Verifier.toml, because nargo prove overwrites Verifier.toml
-# Note: This test should fail, because this y takes the value of x, provided in Prover.toml,
-#  rather than the value of y in Prover.toml 
+# Note: This test should pass, because this y takes the value of y, provided in Prover.toml,
```

Comment:
I was going to suggest that we drop the comment here, but on a second thought it's fine. Keep it.